trajectory_msgs/JointTrajectory old_traj
---
trajectory_msgs/JointTrajectory new_traj
